resources/definitions/fdmprinter.def.json

```diff
@@ -4565,7 +4565,17 @@
                     "type": "bool",
                     "default_value": false,
                     "settable_per_mesh": false,
-                    "settable_per_extruder": false
+                    "settable_per_extruder": false,
+                    "children":
+                    {
+                        "smooth_spiralized_contours":
```

Comment:
BagelOrb has fixed this since then: https://github.com/Ultimaker/Cura/commit/4abcc8b3f15d7a3d4aa3e0ace2a9b4d8c0bcb46c